GNUmakefile

```diff
@@ -113,7 +109,23 @@ install-man: man/bemenu.1 man/bemenu-run.1
 	mkdir -p "$(DESTDIR)$(PREFIX)$(mandir)"
 	cp $^ "$(DESTDIR)$(PREFIX)$(mandir)"
 
-install: install-pkgconfig install-include install-lib-symlinks install-renderers install-bins install-man
+install-renderers: install-curses install-wayland install-x11
+
+install-curses: curses
+	install -d "$(DESTDIR)$(PREFIX)$(libdir)/bemenu"
```

Comment:
These should probably be `mkdir -p` and `cp` as is the case elsewhere.  However, while `install` is not POSIX, it appears that `install -d`  and `install` are both "de facto portable" across Linux, BSD and MacOS.

Reply:
I changed these already in separate commits after noticing :)